configs/e2e_mask_rcnn_R_50_FPN_1x.yaml

```diff
@@ -30,7 +30,8 @@ MODEL:
     SHARE_BOX_FEATURE_EXTRACTOR: False
   MASK_ON: True
 DATASETS:
-  TRAIN: ("coco_2014_train", "coco_2014_valminusminival")
+  TRAIN: ("coco_2014_train",)
```

Comment:
Can you revert this change? All the models have been trained using the new `coco_2017train` dataset, which corresponds to `coco_2014_train` + `coco_2014_valminusminival`. If you want to evaluate at every N iterations, you could do it on the `coco_2014_minival`?

Reply:
I've reverted it and created another config file where the number of iterations for validation specified:
https://github.com/facebookresearch/maskrcnn-benchmark/pull/828/files#diff-4dd26a63ac00a49aeb10985800d7f21c